var/spack/repos/builtin/packages/libcatalyst/package.py

```diff
@@ -36,3 +46,28 @@ def cmake_args(self):
         ]
 
         return args
+
+    def smoke_test(self):
+        spec = self.spec
+        testdir = "smoke_test_build"
+
+        with working_dir(testdir, create=True):
+            cmake = Executable(spec["cmake"].prefix.bin.cmake)
+            cmakeExampleDir = join_path(self.stage.source_path, "examples")
+            catalystPath = join_path(spec["libcatalyst"].prefix.lib.cmake, "catalyst-2.0")
```

Comment:
The examples are failing to use this directory, just pass the prefix through `CMAKE_PREFIX_PATH` and let CMake search inside there.

I think if there's an external conduit, that prefix should be passed as well.